submissions/YuliiaHordiichuk/memory-game/index.js

```diff
@@ -0,0 +1,91 @@
+class Game {
+    constructor() {
+        this.cardsContainer = document.querySelector('.cards'); 
```

Comment:
Let's check what is really needed as a class member and what can be saved as a constant outside of class